### ISSUE TITLE
Modalités de prise non similaires

### ISSUE THREAD

**Issue by iherveOCSIN** [CARA-ch/pmp-issues#22]

**Comment by iherveOCSIN** [comment]
![Image](https://github.com/CARA-ch/pmp-issues/assets/158463876/0a781462-980d-45fe-b35d-16c2d96ba8cf)
![Image](https://github.com/CARA-ch/pmp-issues/assets/158463876/73838d63-9198-4662-bc0e-09737c259e28)

**Comment by iherveOCSIN** [comment]
historique de la question: https://github.com/CARA-ch/ch-emed-epr/issues/38

**Comment by iherveOCSIN** [comment]
@dvribeira va transmettre l'information à @sep-proactive, mais un mapping général devra être établi entre tous les intégrateurs actuels

**Comment by dejajs** [comment]
la colonne de "Modalités de Prise" est la somme de plusieurs paramètres qui on été définis par le métier en interne.
J'observe que dans le portail patient l'information affiché sous "Modalités de Prise" est la valeur du paramètre pmlc.medicationStatement.patientInstructions.
Nous aurons un rdv à ce sujet le lundi 22 avril avec Laure-Zoe, Javier Portela et une autre responsable pour définir où _patientInstructions_ sera mappé dans notre modèle et où l'afficher dans la vue récapitulative.
Ce bug peut rester ouvert en attendant la décision prise.

**Comment by iherveOCSIN** [comment]
@dejajs as-tu du nouveau suite au rdv que tu as eu avec LZK et Javier ? merci

**Comment by dejajs** [comment]
oui, on a trouvé une solution. Je fermerai le ticket jira dès que je l'aurai implémentée.
